app/javascript/packs/containers/NavigationBar/CategoryContainer.jsx

```diff
@@ -0,0 +1,30 @@
+import {Link} from 'react-router-dom'
```

Comment:
Can you change your editor preferences to two spaces (at least for JS) and make sure this applies to all files in your push?